samples/net/dhcpv4_client/README.rst

```diff
@@ -18,28 +18,15 @@ Requirements
 Building and Running
 ********************
 
-QEMU x86
-========
+Running DHCPv4 client in Linux Host
+===================================
 
 These are instructions for how to use this sample application using
 QEMU on a Linux host to negotiate IP address from DHCPv4 server running
 on Linux host.
 
-Follow readme from:
-
-    Run 'loop_socat' and 'loop-slip-tap' scripts from net-tools.
-
-    https://github.com/zephyrproject-rtos/net-tools
-
-.. code-block:: console
-
-    $ ./loop_socat.sh
-
-In another window:
-
-.. code-block:: console
-
-    $ sudo ./loop-slip-tap.sh
+To use QEMU for testing, follow the :ref:`Networking with Qemu
```

Comment:
Qemu should be spelled QEMU.  If you just use :ref:`networking_with_qemu` it will place the title of that label in the generated output link (which is "Networking with QEMU").  You can fix all the references you added this way.